### PR TITLE
add docs for first 2 fn

### LINKED ISSUE #6: Function register courses and record grades

### PR DESCRIPTION
**Pull request by jianructose**

### CODE REVIEW

File: R/frgtracker.R [hunk 1]
```diff
@@ -1,11 +1,38 @@
-# function1 start
 
 
-# function1 end
+# register_courses start
 
-# function2 start
+#' Read and store the input dataframe as a DataFrame.
```
Comment (jraza19) [inline R/frgtracker.R]: Since we are in R now - we dont have to have Dataframe with a capital.

File: R/frgtracker.R [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
+#'
+#' @param df dataframe: A tidy dataframe:
```
Comment (jraza19) [inline R/frgtracker.R]: each argument needs its own "@param". The syntax is also different  - in R we dont start with the type. We usually describe the type in the description itself. please look at some examples : https://kbroman.org/pkg_primer/pages/docs.html

File: R/frgtracker.R [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
+  print('Placeholder')
+}
+
+# register_courses end
+
+# record_grades start
+
+#'Record grades for students for a specified course and its assessments.
+#'
+#' @param df DataFrame: A tidy dataframe
```
Comment (jraza19) [inline R/frgtracker.R]: see above comments!

**Review by jraza19 — COMMENTED** [review]
@jianructose thank you for this! Just take a look at some specific details

**Comment by jianructose** [comment]
#6

**Review by tdkhanhvu — APPROVED** [review]

**Pull request merged**